src/types/pivot.ts

```diff
@@ -166,3 +166,9 @@ export interface DimensionTreeNode {
 }
 
 export type DimensionTree = DimensionTreeNode[];
+
+/** Pivot domain splitted for the domain related to the pivot's rows and columns  */
+export interface PivotColRowDomain {
```

Comment:
On n'a pas déjà ça ?

Reply:
It wasn't merged yet when I coded the PR :) I'll remove it on rebase